### DISCUSSION TITLE
Problem with apache container

### DISCUSSION THREAD

**Discussion by SageSphinx63920** [nextcloud/all-in-one#837] [Questions] [answered]
Hi. I have a problem with my apache container on a fresh install. I used the docs to setup the cloud containers but the logs of the apache container only show:
```
nc: getaddrinfo for host "nextcloud-aio-nextcloud" port 9000: Name or service not known
Waiting for Nextcloud to start...
``` and as im a new user i cant really figure out what to do. Can somebody help me with my problem?

**Comment by szaimen** [comment]
Hi, it may be this problem: https://github.com/nextcloud/all-in-one#how-to-resolve-firewall-problems-with-fedora-linux-rhel-os-centos-suse-linux-and-others

**Comment by SageSphinx63920** [comment]
Im not using any firewall at all for now. I have ufw installed and configured but i disabled it for testing purposes and im connection over localhost to the containers.

Infos about my system:
OS: Ubuntu 22.04 LTS aarch64
Host: Raspberry Pi 4 Model B Rev 1.1(4GB)

**Comment by szaimen** [comment]
Can you post the logs of the Nextcloud container?

**Reply by SageSphinx63920** [reply]
Which one?

**Reply by szaimen** [reply]
The nextcloud container

**Reply by SageSphinx63920** [reply]
![grafik](https://user-images.githubusercontent.com/69404127/174623483-4583d517-be34-4f42-b40f-f7c68f3bad13.png)
there are some of them

**Reply by szaimen** [reply]
nextcloud-aio-nextcloud

**Reply by SageSphinx63920** [reply]
now i cant even connect to the web page. Firefox returns the error "SSL_ERROR_RX_RECORD_TOO_LONG". I even restarted 2 times.

**Reply by SageSphinx63920** [reply]
Sorry im new in docker. I found the logs command. Here is the log https://bin.sageee.xyz/umoruwazad.sql

**Reply by szaimen** [reply]
does restarting all containers from the AIO interface solve this problem?

**Reply by SageSphinx63920** [reply]
After some more AIO container restarts i can access the panel again but my  apacha container does not start like yesterday with the same error.

**Reply by SageSphinx63920** [reply]
any ideas? .-.

**Reply by SageSphinx63920** [reply]
may i should say my nextcloud isnt accessible with the url i've chosen in the setup. My DDNS is a bit buggy rn. Im connecting over localhost/my local network to setup everything right now

**Reply by SageSphinx63920** [reply]
Even with direct ip link in my cloudflare... The container wont start

**Reply by szaimen** [reply]
You need to have a look at the logs...

**Reply by SageSphinx63920** [reply]
i dont understand?

**Reply by szaimen** [reply]
I cannot help you without the logs... 
`nc: getaddrinfo for host "nextcloud-aio-nextcloud" port 9000: Name or service not known` is not helping because this seems to be an issue with docker itself...

**Reply by SageSphinx63920** [reply]
which logs do you need?

**Reply by szaimen** [reply]
lets have a look at the nextcloud-aio-apache logs again...

**Reply by SageSphinx63920** [reply]
the problem is that `nc: getaddrinfo for host "nextcloud-aio-nextcloud" port 9000: Name or service not known`  is my only line in the logs if i type in `sudo docker logs <uuid>`

**Reply by SageSphinx63920** [reply]
at least for the apache container. The logs from the main container are here: https://bin.sageee.xyz/umoruwazad.sql

**Reply by szaimen** [reply]
> the problem is that `nc: getaddrinfo for host "nextcloud-aio-nextcloud" port 9000: Name or service not known` is my only line in the logs if i type in `sudo docker logs <uuid>`

Not even repeated?

**Reply by SageSphinx63920** [reply]
it does repeat. buts is the only thing the log says but like 1000 times

**Reply by szaimen** [reply]
I see. So somehow the dns resolution does not work for that one container. But I cannot reproduce this unfortunately... My best guess is that this is something within docker that fails for you...

**Reply by SageSphinx63920** [reply]
> I see. So somehow the dns resolution does not work for that one container. But I cannot reproduce this unfortunately... My best guess is that this is something within docker that fails for you...

I'm trying to set the default dns server to my local router instead of google's and cloudflare's servers

**Reply by SageSphinx63920** [reply]
I changed the /etc/default docker file and the etc/hosts. Nothing fixes the bug

**Comment by SageSphinx63920** [comment, marked as answer]
Hey,
after some more tests I decided to completely reset my raspberry-pi! I must reinstall everything, because reinstalling the container didn't work...
I changed the url to my ddns url instead of the cname cloud flare record. Its working now.
Thx for the help!

**Reply by szaimen** [reply]
Great to hear that you were able to solve the issue!
